platformio.ini

```diff
@@ -61,7 +61,7 @@ upload_port = 192.168.1.152
 
 
 [debug_flags]
-build_flags               =
+build_flags               = -Wstack-usage=300
```

Comment:
This is only a warning I used to track functions using a lot of stack.
For normal builds it should be placed in comments (leading with semi-colon).

Reply:
Just one question. 

Did you mean. The line should look like

; build_flags = -Wstack-usage=300 

regards Chris

Reply:
Well there should be a line with the empty one and one in comments so I don't have to look for the correct compiler flag again if I would like to test it.

With the warning set to `stack-usage=300`, you will get quite a lot of warnings when compiling.
Each function that needs 300 bytes or more when being called is triggering a warning.
So that may be very confusing to anyone that doesn't know about this warning.
That's why I want to have it disabled by default and have it available by just removing a comment.
But PlatformIO will fail to build if you refer to a field in some environment which doesn't exist, so if not used you must leave it empty:
`build_flags =`

Yesterday I merged in the way I meant it to be from the mega branch of our repository.